contracts/staking/ValidatorManager.sol

```diff
@@ -53,7 +53,7 @@ abstract contract ValidatorManager is
 
     // solhint-disable ordering
     function _getValidatorManagerStorage()
-        private
+        internal
```

Comment:
would be wary of making this `internal` as it uses assembly to assign slot of value `$`, the same for other contracts

Reply:
Did we make a decision on what we wanted to do here? The churn-tracker PR also makes this function internal in order to access `ValidatorManager` state, but it's only to access a validator's weight. We might be okay adding a separate getter for the validator state.